packages/playground/src/components/k8s_deployment_table.vue

```diff
@@ -15,17 +15,20 @@
         <v-card-title style="color: #ffcc00; font-weight: bold">Failed Deployments</v-card-title>
         <v-divider color="#FFCC00" />
         <v-card-text>
-          <ul>
-            <li v-for="deployment in failedDeployments" :key="deployment.name">
-              {{
-                deployment.nodes.length > 0
-                  ? `${deployment.name} on node${deployment.nodes.length > 1 ? "s" : ""}: ${deployment.nodes.join(
-                      ", ",
-                    )}`
-                  : deployment.name
-              }}
-            </li>
-          </ul>
+          <li v-for="deployment in failedDeployments" :key="deployment.name">
+            {{
+              deployment.nodes.length > 0
+                ? `${deployment.name} on node${deployment.nodes.length > 1 ? "s" : ""}: ${deployment.nodes.join(", ")}`
+                : deployment.name
+            }}
+            with contract id:
+            <template v-if="deployment.contracts && deployment.contracts.length > 0">
```

Comment:
if there is no contracts, the text will be `with contract id: No contracts found.` ?

Reply:
- u're right, it doesn't make sense

Reply:
i guess we need to set the `with contract id` inside the if